packages/client/src/routes/story-pointing/meeting.tsx

```diff
@@ -0,0 +1,83 @@
+import { useLoaderData, useParams } from "react-router-dom";
+import { useState, useEffect } from 'react';
+import { getMeeting } from "../../models/meeting";
+import { getLoggedUser, getUsers } from "../../models/user";
+import useWebSocket from 'react-use-websocket';
+import { StoryPointingAdminControllers } from '../../components/StoryPointingAdminControllers';
+import { StoryPointingPoints } from '../../components/StoryPointingPoints';
+import { StoryPointingVotes } from '../../components/StoryPointingVotes';
+
+export async function loader({ params }) {
+  const user = await getLoggedUser();
+  const users = await getUsers();
+  const meeting = await getMeeting(params.meetingId);
```

Comment:
It will be faster running those in parallel
```suggestion
  const [user, users, meeting] = await Promise.allSettled([
    getLoggedUser(),
    getUsers(),
    getMeeting(params.meetingId)
  ])
```